packages/lib-classifier/src/store/UserProjectPreferencesStore.js

```diff
@@ -89,8 +94,76 @@ const UserProjectPreferencesStore = types
       }
     }
 
-    function updateUPP (changes) {
-      console.log('TODO: UPP PUT request', changes)
+    function * fetchUPPById (id = '') {
```

Comment:
Maybe this should be a method on the UPP model, since it always returns a single resource. Something like `UPP.fetch(id)`.

Reply:
I'm planning on moving it into panoptes.js at a later date.